miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/LibraryDilutionRestController.java

```diff
@@ -89,7 +89,7 @@ public void setDilutionService(LibraryDilutionService dilutionService) {
   @ResponseBody
   public DilutionDto getDilution(@PathVariable Long dilutionId) throws IOException {
     LibraryDilution dilution = dilutionService.get(dilutionId);
-    DilutionDto dilutionDto = Dtos.asMinimalDto(dilution);
+    DilutionDto dilutionDto = Dtos.asDto(dilution, false, false);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "dilutionDto". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)